mu4/scores/view/newscoremodel.cpp

```diff
@@ -79,17 +79,17 @@ void NewScoreModel::setComposer(QString composer)
     emit composerChanged(m_composer);
 }
 
-void NewScoreModel::fillDefault(ScoreInfo& scoreInfo)
+void NewScoreModel::fillDefault(ScoreCreateOptions& scoreOptions)
 {
-    scoreInfo.subtitle = "default subtitle";
-    scoreInfo.poet = "default poet";
-    scoreInfo.copyright = "default copyright";
-    scoreInfo.tempo = 120;
-    scoreInfo.timesigNumerator = 4;
-    scoreInfo.timesigDenominator = 4;
-    scoreInfo.measures = 32;
-    scoreInfo.measureTimesigNumerator = 1;
-    scoreInfo.measureTimesigDenominator = 4;
-    scoreInfo.timesigType = Ms::TimeSigType::NORMAL;
-    scoreInfo.key = Ms::Key::C;
+    scoreOptions.subtitle = "default subtitle";
+    scoreOptions.poet = "default poet";
```

Comment:
I believe the user visible term is "lyricist"

Reply:
And I'd just drop the word "default" from these

Reply:
This is a temporary solution.

Reply:
I suggest adding: `TODO Temporary solution`